libsoftwarecontainer/component-test/softwarecontainerlib_componenttest.cpp

```diff
@@ -365,9 +367,65 @@ TEST_F(SoftwareContainerApp, TestFolderMounting) {
     job3->start();
     ASSERT_EQ(job3->wait(), EXISTENT);
 
+    // Test that we can mount to some place where there is something else
+    // mounted higher up in the hierarchy (in this case, /dev)
+    ASSERT_TRUE(bindMountInContainer("/dev/shm", "/dev/shm", false));
+    auto jobShm = getSc().createFunctionJob([&] () {
+        return isDirectory("/dev/shm") ? EXISTENT : NON_EXISTENT;
+    });
+    jobShm->start();
+    ASSERT_EQ(jobShm->wait(), EXISTENT);
```

Comment:
IIRC we added convenience functions for checking the result of jobs, please use them instead.